index.html

```diff
@@ -9,10 +9,17 @@
 <body>
     <!-- objects for background -->
 
+    <div id="instructions_div">
+        <p id= "instructions_headline">Instructions</p>
+        <p id="instuctions">Today's headline is presented to you in a scrambled form 🧩 You must unscramble the headline word by word, using the box below 📩 There is no way to fail this game, however, you are being timed ⏳, and your score will be presented to you upon completion 🥇 </p>
+        <p id="example"> mlexeap = example</p>
```

Comment:
![Screenshot 2022-05-17 at 16 44 17](https://user-images.githubusercontent.com/8916090/168853348-84ea5bed-94b1-4a77-9744-aeecdb58a0f6.png)

What does this P tag do? I don't believe it is required

Reply:
Just to make extra clear to players what 'scrambled' means :)